action/action-dsl/src/main/groovy/eu/itesla_project/action/dsl/task/ScriptDslTaskExtension.groovy

```diff
@@ -27,7 +28,11 @@ class ScriptDslTaskExtension implements DslTaskExtension, DslConstants {
                 binding.setVariable("computationManager", computationManager)
                 binding.setVariable(SCRIPT_IS_RUNNING, true)
                 try {
+                    closure.owner.delegate.metaClass = null
+                    closure.resolveStrategy = Closure.OWNER_ONLY
                     closure.call()
+                } catch (MissingMethodException e) {
```

Comment:
Rethrow is not needed here because MissingMethodException is already a runtime exception

Reply:
I would like to rethrow a user-friendly message thats tells user dsl extentions task is forbidden in a script tast